src/components/BulkEnrollmentPage/CourseSearchResults.jsx

```diff
@@ -90,6 +91,10 @@ export const BaseCourseSearchResults = (props) => {
       // eslint-disable-next-line react/prop-types
       Cell: ({ value, row }) => <CourseNameCell value={value} row={row} enterpriseSlug={enterpriseSlug} />,
     },
+    {
+      Header: TABLE_HEADERS.partnerName,
+      accessor: 'partners[0].name',
```

Comment:
What is the cell field here and why are we not using it for this column?

Reply:
cell field is only to customize how it's rendered. Data table by default renders as a span I think which suffices for this use case

Reply:
it comes out like this by default 

```
<td role="cell"><span class="pgn__data-table-cell-wrap">edx</span></td>
```